db/schema.rb

```diff
@@ -28,6 +28,8 @@
     t.bigint "account_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "media_type"
```

Comment:
why is this required?

Reply:
Because I am defining (self.inheritance_column = :media_type) this in Media class so that the audio and video can perform their respective function

Reply:
for that you have added a migration, why this has to be changed in schema.rb?

Reply:
Audio and Video have different function to calculate duration_tc, If there is no field to differentiate audio and video how the Media class
 will call the respective functions?